astro/src/components/Navbar.astro

```diff
@@ -0,0 +1,26 @@
+<div class="flex justify-between items-center py-10 px-4">
+  <div class="container-1 flex items-center gap-5">
+    <div class="hamburger-menu"></div>
+    <div class="logo text-xl font-extrabold">BiciRubbate</div>
```

Comment:
Bad! hardcoded strings versus `t()` syntax

Reply:
Is there some kind of logic to follow for naming `t()` function arguments that do not exists in the database yet?

e.g. `pages.homepage.cta`, `pages.homepage.description.bold` and so on nesting components

Reply:
@fede03billy the ideal would be `<page>`.`<component>`. `something`
and something like `_global`.`something` for shared . I dont know if there's a well established pattern for naming convention in this area, but this seems appropriate, especially keeping consistency between filenames, component names and string prefixes